mne/io/cnt/cnt.py

```diff
@@ -459,10 +460,15 @@ class RawCNT(BaseRaw):
         the header are correct, it is probably safer to use a (standard)
         montage. See :func:`mne.channels.make_standard_montage`
 
+    .. note::
+
+        A CNT file can also come from the EEG manufacturer ANT Neuro, in which case the
+        function :func:`mne.io.read_raw_ant` should be used.
```

Comment:
This reminds me, we need to add this to `read_raw`, right?

If so it might be good to add some little check at some point that all `read_raw_*` functions show up in `read_raw` somewhere (maybe with exclusions/exceptions), as it would have caught this potential oversight. Something as simple as reading the source file that contains `read_raw` and the `io.rst` (or wherever we document raw readers), and then making sure all things that match `read_raw_*` in `io.rst` show up in the `read_raw`. I can take a quick stab at this if you want @mscheltienne .

Then there is a separate issue of how to handle the file extension overlap. If we don't already have logic for that in `read_raw` we could use `try/except`-based logic.

Reply:
I forgot that one! I'll add the test as suggested.